tests/test_quadtree.py

```diff
@@ -39,16 +48,20 @@ def test_str_with_particles(self):
         self.assertEqual(actual, expected)
 
     def test_query(self):
+        particle_num = 5
+
         # Create five particles
-        particles = [Particle(f"Test_{i}", Vector2(25, 50), 2, (255, 255, 255)) for i in range(5)]
+        particles = [Particle(f"Player", Vector2(25, 50), 2, (255, 255, 255)) for i in range(particle_num)]
```

Comment:
We create player objects; not particle objects with a string arg stating it's a player

Reply:
As a sidenote; I've just realised this is how I set it up; I'm an idiot and didn't set it up correctly